### DISCUSSION TITLE
Extending API

### DISCUSSION THREAD

**Discussion by samip5** [postalserver/postal#2367] [Feature suggestions]
I would love to see more than just email sending to be possible via API.

Things I would love to see:

- Create servers & add domains to them (with corresponding functionality as in frontend)
- Create routes and destinations

**Comment by samip5** [comment]
Oh, this can be deleted/closed as it's a duplicate of https://github.com/postalserver/postal/discussions/2362.

**Comment by arthurzenika** [comment]
In our usecase of postal, we have a component of our software that does this though SQL, so an API for this would be greatly appreciated and improve the security of the architecture.
